internal/utils/slices.go

```diff
@@ -0,0 +1,9 @@
+package utils
+
+// AppendRefs returns append(s, &v[0], &v[1], ...).
+func AppendRefs[T any](s []any, v []T) []any {
```

Comment:
is this still needed with golang slices generic stuff ?

Reply:
Just went through the package docs. Nothing there does it compactly.